ginga/misc/plugins/TVMask.py

```diff
@@ -0,0 +1,447 @@
+"""Non-interactive masks display local plugin for Ginga."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ginga.util.six import itervalues
+
+# STDLIB
+import re
+import os
+
+# THIRD-PARTY
+import numpy as np
+from astropy.io import fits
```

Comment:
This bypasses `ginga.util.io_fits` but it is only used to load boolean masks for this plugin. Is this a problem for Ginga?